supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiBackendLifecycleActorFactory.scala

```diff
@@ -82,4 +90,48 @@ abstract class PipelinesApiBackendLifecycleActorFactory(override val name: Strin
 object PipelinesApiBackendLifecycleActorFactory {
   val preemptionCountKey = "PreemptionCount"
   val unexpectedRetryCountKey = "UnexpectedRetryCount"
+
+  private val logger: Logger = LoggerFactory.getLogger(getClass)
```

Comment:
Is there not a `StrictLogger` we could use?

Reply:
I can mix the`StrictLogging` [trait](https://github.com/lightbend/scala-logging/blob/2c238c9347006b7bbaaa889289ff6eeaa1df102a/src/main/scala/com/typesafe/scalalogging/Logging.scala#L22) into the `object`. tbh it feels a little weird as this is a companion object to the Actor factory as opposed to an Actor, but if Akka wanted to prevent us from doing this they probably would have [self-typed](https://docs.scala-lang.org/tour/self-types.html) it... 🙂